octobot_trading/personal_data/trades/trade.py

```diff
@@ -59,12 +64,14 @@ def update_from_order(self, order, creation_time=0, canceled_time=0, executed_ti
         self.status = order.status
         self.fee = order.fee
         self.trade_id = order.order_id
+        self.origin_order_id = order.order_id
         self.simulated = order.simulated
         self.side = order.side
         self.creation_time = order.creation_time if order.creation_time > 0 else creation_time
         self.canceled_time = order.canceled_time if order.canceled_time > 0 else canceled_time
         self.executed_time = order.executed_time if order.executed_time > 0 else executed_time
         self.symbol = order.symbol
+        self.is_closing_order = order.is_closed() or order.is_filled() or order.is_cancelled()
```

Comment:
How can a trade exists if it's not linked to a closed, filled or cancelled order ?

Reply:
On ccxt, a partially filled order is "open" => when we will properly handle partially filled orders, we should create trades for each actual trades of an order, even if the order is still open (in this case being partially filled)

Reply:
Ok I see, good idea :+1: